packages/cli/src/templates/snow/hooks/write-trellis-context.py

```
@@ -0,0 +1,585 @@
+
+def _find_repo_root(start: Path) -> Path:
+    cur = start.resolve()
+    for candidate in [cur, *cur.parents]:
+        if (candidate / ".trellis").is_dir():
+            return candidate
+    return cur
+
+
+def _run(cmd: list[str], cwd: Path) -> str:
+    try:
+        completed = subprocess.run(
```

Ruff (0.15.21)

[error] `subprocess` call: check for execution of untrusted input

(S603)